src/constructs/autoscaling/user-data.ts

```diff
@@ -39,6 +44,27 @@ export interface GuUserDataProps {
 export class GuUserData {
   private readonly _userData: UserData;
 
+  /**
+   * A helper function to make user data strings easier to write.
+   *
+   * Example usage:
+   * ```typescript
+   * const additionalStatements = GuUserData.stripMargin`
+   *   |mkdir /etc/gu
+   *   |cat > /etc/gu/my-application.conf <<-'EOF'
+   *   |  include "application"
+   *   |EOF
+   *   |`;
```

Comment:
I was initially confused about how `stripMargin` worked, but I think this description is good enough to allow library users to use it without confusion. I'm still confused about how the ``` stripMargin`string foo bar baz` ``` syntax works but covered that in an earlier comment.